visualization/app.py

```diff
@@ -0,0 +1,147 @@
+"""
+Create Plotly Dash Visualization to map the cleaned data
+
+Author(s): Aïcha Camara
+"""
+import plotly.graph_objects as go
```

Comment:
This directory isn't in the application.

Reply:
@necabotheking Because idk what that means

Reply:
You added a whole other `app.py` file in a folder that didn't exist prior.